test/common/lcov.py

```diff
@@ -171,6 +171,8 @@ def get_distfile(url, dist_map, webpack_name):
         else:
             sys.stderr.write(f"SKIP {url} -> {path}\n")
 
+    return None
```

Comment:
Same with the comment above.  This is effectively the "error case" and should be treated like that, instead of indenting the "normal case".